p11-kit/iter.c

```diff
@@ -636,6 +642,11 @@ move_next_session (P11KitIter *iter)
 		if (rv != CKR_OK || !p11_match_uri_token_info (&iter->match_token, &iter->token_info))
```

Comment:
the indentation seems to be a bit messed up in this file

Reply:
I've wrapped long lines (if you mean that).